tests/PHPStan/Analyser/data/cli-globals.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace CliGlobals;
+
+\PHPStan\Testing\assertType('int<1, max>', $argc);
+\PHPStan\Testing\assertType('non-empty-array<int, string>', $argv);
+
+function f() {
+	\PHPStan\Testing\assertType('*ERROR*', $argc);
+	\PHPStan\Testing\assertType('*ERROR*', $argv);
+}
+
+function g($argc, $argv) {
+	\PHPStan\Testing\assertType('mixed', $argc);
+	\PHPStan\Testing\assertType('mixed', $argv);
+}
+
+function h() {
+	global $argc, $argv;
+	\PHPStan\Testing\assertType('mixed', $argc); // should be int<1, max>
+	\PHPStan\Testing\assertType('mixed', $argv); // should be non-empty-array<int, string>
```

Comment:
tried several things, but in the end wasn't able to identify this variable-scope.

therefore left it `mixed` for now